### DISCUSSION TITLE
Bun http2

### DISCUSSION THREAD

**Discussion by Fnux8890** [honojs/hono#2329] [Q&A] [answered]
In the docs it's stated that http2 can be used but its node specific.
Im curious as to how it could be done in bun? 🙏

**Comment by yusukebe** [comment, marked as answer]
Hi @Fnux8890 

Maybe Bun doesn't support HTTP/2.

**Reply by Fnux8890** [reply]
It should as of [1.0.13](https://bun.sh/blog/bun-v1.0.13)
Im not sure how to implement it with hono tho 🤔

**Reply by yusukebe** [reply]
That's a client-side function. It's not for a server-side use case.

**Reply by Fnux8890** [reply]
Ah I see 👍
Not to sure what the difference would be but I could look into that.

Thanks for the responses btw💪
